lib/workers/global/config/parse/file.ts

```diff
@@ -81,3 +89,26 @@ export async function getConfig(env: NodeJS.ProcessEnv): Promise<AllConfig> {
   }
   return config;
 }
+
+export async function deleteConfigFile(env: NodeJS.ProcessEnv): Promise<void> {
+  const configFile = env.RENOVATE_CONFIG_FILE;
+
+  if (is.undefined(configFile) || is.emptyStringOrWhitespace(configFile)) {
+    return;
+  }
+
+  if (!(await fs.pathExists(configFile))) {
+    return;
+  }
+
+  if (env.RENOVATE_X_DELETE_CONFIG_FILE !== 'true') {
+    return;
+  }
```

Comment:
Seems to be duplicate logic

Reply:
yes, it is international. 
i dont want to invoke the function without any explicit safe checks, nor i want this function to delete the config without safe checks either as it is exported (for testing).

we can  maybe rename the function to `tryDeleteConfigFile` and remove the external safe checks.